flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["aarch64", "x86_64"],
+    "skip-appstream-check": true
```

Comment:
Appdata should be fixed instead of disabling the check, and if it doesn't already, it should mention it's an LTR version.